runtime/moonbase/src/lib.rs

```diff
@@ -307,6 +317,12 @@ impl FeeCalculator for FixedGasPrice {
 	}
 }
 
+/// Parameterized slow adjusting fee updated based on
+/// https://w3f-research.readthedocs.io/en/latest/polkadot/
+///                                              Token%20Economics.html#-2.-slow-adjusting-mechanism
```

Comment:
did cargo fmt do this indent?

Reply:
No, this was me. Do we have a way to squelch `editorconfig` on a case-by-case basis...?

Reply:
lol, this link is a 404 anyway :laughing:

Reply:
This is probably the same (or updated) document: https://w3f-research.readthedocs.io/en/latest/polkadot/overview/2-token-economics.html